graylog2-server/src/main/java/org/graylog2/indexer/indices/Indices.java

```diff
@@ -111,17 +112,17 @@
     private static final Logger LOG = LoggerFactory.getLogger(Indices.class);
     private static final String REOPENED_INDEX_SETTING = "graylog2_reopened";
 
+    public static final String ALL_INDEX_PREFIX = "*";
```

Comment:
Good catch, changed that so the `getAllMessageFields` method takes an array of index prefix wildcards.